### ISSUE TITLE
Crop dialog only fits image to window width

### ISSUE THREAD

**Issue by LRN** [rquast/swingsane#5]
If an image being cropped has portrait orientation (lot of height, not much width; this is the default for the scanner i have), the crop dialog will still display the image by fitting its width to the window. Since most PC screens have landscape orientation, the crop window (when maximized) tends to be very wide, but not very tall. As a result, the image looks as if its top and bottom parts are clipped.
The crop window provides no way to zoom out or drag the view, no scrollbars and no indication of what's happening.
At first i thought it's a bug. Then i rotated the image before cropping it, and that fixed the problem (the image got landscape orientation, so fitting it by width fits it correctly). After that i've realized that instead of maximizing the crop window, i can resize it to be narrower, and that fixed the problem too. Still, it's not immediately obvious, and thus very confusing.

Unrelated: swingsane doesn't support the sane options for my scanner, but that's probably the expected behaviour for unsupported models. Where do i send the option xml files? The scanner.xml page of the wiki is empty.

**Comment by rquast** [comment]
Yes, I'd like to rework that entire area of the program actually. I originally tried to find a library that would do all that but didn't find anything that great. Feel free to try this bit out with a third party library or enhance the existing code.
